libs/react/src/lib/form/input/input.tsx

```diff
@@ -65,56 +66,62 @@ export const Input = ({
     setLocalValue(value)
   }, [value])
 
-  const localOnChange = useCallback(
+  const handleChange = useCallback(
     (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Updated how the caret position is selected, did not work as it was before. Now we always keep the same caret position as earlier, but if the formatting ads or removes chars, then we move the caret the same number of steps as chars added. I have tested it with two of our formatters, might not work for all though.

Formatters:
formatAmount 
formatPersonalIdentificationNumber 
https://github.sebank.se/toconsume/consumer-loan-mfe/blob/develop/src/common/helpers/formatters.ts